### PR TITLE
ultradns: cleanup

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

File: builtin/providers/ultradns/config.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func (c *Config) Client() (*udnssdk.Client, error) {
 		return nil, fmt.Errorf("Error setting up client: %s", err)
 	}
 
-	log.Printf("[INFO] UltraDNS Client configured for user: %s", client.Username)
+	log.Printf("[INFO] UltraDNS Client configured for user: %s", c.Username)
```
Comment (scottcunningham) [inline builtin/providers/ultradns/config.go:25]: I can't see anywhere where `client` is renamed to `c`, is this change still good? I'm probably missing something.

**Comment by scottcunningham** [comment]
I think this looks fine so 👍

**Comment by thejandroman** [comment]
🎱

File: builtin/providers/ultradns/config.go (hunk 1 above)
Comment (josephholsten) [inline builtin/providers/ultradns/config.go:25]: `c` is the `*Config`. By switching to an `oauth2.TokenSource` we'd have needed to dig deep into the actual `client` state to access the username, so grabbing it from a struct owned by the provider seemed more stable.

**Pull request merged**